.github/workflows/release-desktop.yml

```
@@ -136,6 +146,7 @@ jobs:
         run: mv Bitwarden-${{ env.PKG_VERSION }}-universal.pkg Bitwarden-${{ env.PKG_VERSION }}-universal.pkg.archive
 
       - name: Set staged rollout percentage
+        if: ${{ inputs.electron_publish }}
```

```suggestion
        if: ${{ github.event.inputs.electron_publish }}
```

Looks like the rest of the inputs use `github.event.inputs.*`

---

Oh, you are right. I took a hint from this: https://github.com/bitwarden/clients/blob/master/.github/workflows/release-desktop.yml#L234 But there are no curly braces there.